extensions/pkg/controller/bastion/configvalidator.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // ConfigValidator validates the provider config of bastion resource with the cloud provider.
 type ConfigValidator interface {
-	// Validate validates the provider config of the given infrastructure resource with the cloud provider which used by Bastion.
+	// Validate validates the provider config bastion and cluster resource is given with the cloud provider used by Bastion.
```

Comment:
Please correct this sentence, it does not make much sense

Reply:
sorry wrong copy-paste here change back https://github.com/gardener/gardener/commit/3b823f3f48a1660dca9f0020965b0bfdb5709cd2